.cargo/config.toml

```diff
@@ -6,3 +6,10 @@
 quick = "make quick"
 tidy = "make tidy"
 clippy-all = "clippy --all-features --all-targets -- -D warnings"
+
+[unstable]
+# Use `cargo +nightly update` to update MSRV-aware
+msrv-policy = true
+
+[resolver]
+something-like-precedence = 'something-like-rust-version'
```

Comment:
nit: please link to docs for these fields (I know they're primarily documented in PRs right now)